### PR TITLE
Update linting.md

### PR DESCRIPTION
**Pull request by Riyan911**
Added target="_blank" for all links under Knowledge Check

## Because

Added html attributes so that links are opened in a new tab.

## This PR

•Added target = "_blank" for all <a> elements under Knowledge Check.

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: javascript/javascript_in_the_real_world/linting.md [hunk 1]
```diff
@@ -52,6 +52,6 @@ Another way to address the conflict is to use `eslint-plugin-prettier`. It lets
 
 This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
 
-- <a class="knowledge-check-link" href="https://mikecavaliere.com/javascript-linting-what-developers-need-to-know/">What is linting?</a>
-- <a class="knowledge-check-link" href="https://mikecavaliere.com/javascript-linting-what-developers-need-to-know/">Which problems can linting prevent?</a>
-- <a class="knowledge-check-link" href="https://www.youtube.com/watch?v=hkfBvpEfWdA">Why should you use Prettier?</a>
+- <a target="_blank" class="knowledge-check-link" href="https://mikecavaliere.com/javascript-linting-what-developers-need-to-know/">What is linting?</a>
```
Comment (KevinMulhern) [inline javascript/javascript_in_the_real_world/linting.md:55]: Nice spot with these @Riyan911 

We've been using markdown links for any recently added knowledge links, can we do that here as well please? 
Our lesson parser will automatically add a `target="_blank"` to any markdown links.

Example:
```suggestion
- [What is linting?](https://mikecavaliere.com/javascript-linting-what-developers-need-to-know)

**Review by KevinMulhern — CHANGES_REQUESTED** [review]

File: javascript/javascript_in_the_real_world/linting.md (hunk 1 above)
Comment (Riyan911) [reply javascript/javascript_in_the_real_world/linting.md:55]: Sure 😀. I've been trying to look for projects to contribute and came across these when I was learning from The Javascript section, so I thought to create a pull request. Thanks for the early reply though.

**Comment by Riyan911** [comment]
I made the necessary changes as requested and committed the changes. What to do next?

File: javascript/javascript_in_the_real_world/linting.md [hunk 2]
```diff
@@ -52,6 +52,6 @@ Another way to address the conflict is to use `eslint-plugin-prettier`. It lets
 
 This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
 
-- <a class="knowledge-check-link" href="https://mikecavaliere.com/javascript-linting-what-developers-need-to-know/">What is linting?</a>
-- <a class="knowledge-check-link" href="https://mikecavaliere.com/javascript-linting-what-developers-need-to-know/">Which problems can linting prevent?</a>
-- <a class="knowledge-check-link" href="https://www.youtube.com/watch?v=hkfBvpEfWdA">Why should you use Prettier?</a>
+- [What is linting](https://mikecavaliere.com/javascript-linting-what-developers-need-to-know/)
```
Comment (KevinMulhern) [inline javascript/javascript_in_the_real_world/linting.md:55]: This is looking great [on the preview](https://www.theodinproject.com/lessons/preview?uuid=138a5ff5-93f2-47ba-9cf9-7c2610dbb443), but theres just one small typo. We need a question mark here:

```suggestion
- [What is linting?](https://mikecavaliere.com/javascript-linting-what-developers-need-to-know/)
```

**Review by KevinMulhern — CHANGES_REQUESTED** [review]

File: javascript/javascript_in_the_real_world/linting.md (hunk 2 above)
Comment (Riyan911) [reply javascript/javascript_in_the_real_world/linting.md]: Done! So excited. This is my first time contributing to open source.

**Pull request merged**